nomad/structs/structs.go

```diff
@@ -5136,6 +5136,12 @@ func (u *UpdateStrategy) IsEmpty() bool {
 		return true
 	}
 
+	// When the Job is transformed from api to struct, the Update Strategy block is
+	// copied into the existing task groups, the only things that are passed along
+	// are MaxParallel and Stagger, because they are enforced at job level.
+	// That is why checking if MaxParallel is cero is enough to know if the
```

Comment:
thanks for adding this!
```suggestion
	// That is why checking if MaxParallel is zero is enough to know if the
```